### DISCUSSION TITLE
Option to transfer skin layout between selected skins

### DISCUSSION THREAD

**Discussion by SevenChords** [ppy/osu#19892] [Ideas]
I use multiple skins, mostly depending on how i feel like, and if i want to have the exact same skin layout shared between them right now i would have to either be able to precicely place elements in the correct sport for each skin or export all skins i want to share the same layout after setting it on one to transfer the main hud layout json between all the skins.

It would be great if this process could be simplified by having an in game option to copy the layout from one skin to one or multiple other skins.

**Comment by Terratype** [comment]
I've been needing this exact thing for so long. It's especially infuriating when I switched from Argon to Argon pro, a near identical skin, but having it reset my whole layout just got me to never use it.

**Reply by Terratype** [reply]
Actually another idea I had is an option to keep your current layout no matter the skin, so that even downloading a new skin you would still have your layout as you'd like it.

**Comment by Terochi** [comment]
Another roundabout way to fix this issue would be to just add copy pasting. That would however need the implementation of some sort of "global clipboard" if we don't want to expose the copied data to the user.

**Reply by peppy** [reply]
https://github.com/ppy/osu/pull/22650
